bin/crew

```diff
@@ -772,9 +772,8 @@ def download
       Dir.mkdir @extract_dir
       Dir.chdir @extract_dir do
         if @pkg.git_branch.to_s.empty?
-          system 'git init'
-          system 'git config advice.detachedHead false'
-          system 'git config init.defaultBranch master'
+          system 'git init --quiet', exception: true # use --quiet to suppress default branch name warning
```

Comment:
I'm not a fan of quiet.  Hiding stuff is not what I prefer.

Reply:
There is no other way to hide this:
```
Performing pre-flight checks...
No precompiled binary available for your platform, downloading source...
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
```

Reply:
We should probably start setting `git config --global init.defaultBranch main` since that's becoming the new norm...

Reply:
Can we get rid of line 775 and merge this?
The git config line in line 776 should in theory make this message go away, no?